modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/test/components/instance-list-page/InstanceListPageItem.es.js

```diff
@@ -468,6 +468,33 @@ describe('The InstanceListPageItem instance checkbox component should', () => {
 		taskNames: ['Review'],
 	};
 
+	test('Disable the checkbox when the id is different of the userId', () => {
+		const {container} = render(
+			<Table.Item {...{...instance, id: 12345}} />,
```

Comment:
Hi @seltong , I dont see a problem with this... 
But In this line and 487 line, you can do it like this: `<Table.Item {...instance} id={12345} />`
Remember that "explicit is better than implicit".